wp-content/themes/core/assets/css/src/theme/forms/_mixins.pcss

```diff
@@ -22,12 +22,12 @@
  * ----------------------------------------------------------------------------- */
 
 @define-mixin form-control-helper {
-	margin-top: var(--form-control-helper-margin-top);
-	padding: var(--form-control-helper-padding);
-	color: var(--form-control-helper-color);
+	margin-top: 8px;
+	padding: 0 8px;
+	color: var(--form-color-secondary);
 	letter-spacing: var(--form-control-helper-letter-spacing);
 	text-transform: var(--form-control-helper-text-transform);
-	font: var(--form-control-helper-font-style) var(--form-control-helper-font-weight) var(--form-control-helper-font-size) / var(--form-control-helper-font-line-height) var(--form-control-helper-font-family);
+	font: normal var(--form-control-helper-font-weight) var(--form-control-helper-font-size) / var(--form-control-helper-font-line-height) var(--form-control-helper-font-family);
```

Comment:
The shorthand font declaration here doesn't always work x-browser. This is due to the keyword `inherit` occasionally being used for line-height, which is [invalid per the spec](https://stackoverflow.com/a/5518592) for shorthand properties, like `font`. The resolution I have generally used on projects it to just break up the shorthand `font` declaration into separate properties for each.  This is true throughout the forms sink (and SquareOne in general).

Reply:
good insight, I can break those up across the board for forms